src/Components/Atbash/AtbashInput.js

```diff
@@ -1,14 +1,37 @@
 import React, { Component } from "react";
 import LetterBox from "../LetterBox.js";
-import LetterBoxesWithArrows from "../LetterBoxesWithArrows/LetterBoxesWithArrows.js"
-import GetInput from '../Input/Input.js'
+import GetInput from '../Input/Input.js';
+import { atbashEncode } from '../../atbashEncode.js';
+import Anime, {anime} from 'react-anime';
 
 class Atbash extends Component {
   state = {
     inputStr: null,
     outputStr: null,
   };
-  
+
+  constructor(props) {
+    super(props);
+    this.timeouts = [ 0 ]
+    this.decrypted = [false, false, false]
+    this.state = {
+      animateDecrypt: [false, false, false],
+      // animateDecrypt2: false,
+      // animateDecrypt3: true,
+      messages: [
+        atbashEncode("[ Getting the hang of this? ]"),
+        atbashEncode("[ ...Or going bananas? ]"),
+        atbashEncode("[ Hey now, you're a rock star! ]")
+      ]
+    }
+  }
+
+  componentWillUnmount() {
+    for(let i = 0; i < this.timeouts.length; i++) {
+      this.timeouts[i] = clearTimeout()
+    }
+  }
```

Comment:
The way things are set up right now, your timeouts roll over, rather than being cleared. Assuming an `Array`, we can fix this:

```suggestion
  componentWillUnmount() {
    this.timeouts.forEach(t => clearTimeout(t));
  }
```